devicetypes/smartthings/rgbw-light.src/rgbw-light.groovy

```diff
@@ -27,15 +27,60 @@ metadata {
 		capability "Sensor"
 		capability "Health Check"
 
-		command "reset"
-
-		fingerprint inClusters: "0x26,0x33"
-		fingerprint inClusters: "0x33"
+		/*
+		 * Relevant device types:
+		 *
+		 * * 0x11 GENERIC_TYPE_SWITCH_MULTILEVEL
+		 * * 0x01 SPECIFIC_TYPE_POWER_SWITCH_MULTILEVEL
+		 * * 0x02 SPECIFIC_TYPE_COLOR_TUNABLE_MULTILEVEL
+		 *
+		 * Plausible command classes we might see in a color light bulb:
+		 *
+		 * 0x98 COMMAND_CLASS_SECURITY
+		 * 0x5E COMMAND_CLASS_ZWAVEPLUS_INFO_V2
+		 * 0x20 COMMAND_CLASS_BASIC
+		 * 0x26 COMMAND_CLASS_SWITCH_MULTILEVEL
+		 * 0X27 COMMAND_CLASS_SWITCH_ALL
+		 * 0x33 COMMAND_CLASS_SWITCH_COLOR
+		 * 0x70 COMMAND_CLASS_CONFIGURATION
+		 * 0x73 COMMAND_CLASS_POWERLEVEL
+		 *
+		 * Here are the command classes used by this driver that we can fingerprint against:
+		 *
+		 * * 0x26 COMMAND_CLASS_SWITCH_MULTILEVEL -> yes, it is dimmable
+		 * * 0x33 COMMAND_CLASS_SWITCH_COLOR -> yes, it has color control
+		 */
+
+		// dimmable, color control
+		fingerprint inClusters: "0x26,0x33", deviceJoinName: "Z-Wave RGBW Bulb"
+
+		// GENERIC_TYPE_SWITCH_MULTILEVEL:SPECIFIC_TYPE_POWER_SWITCH_MULTILEVEL
+		// dimmable, color control
+		fingerprint deviceId: "0x1101", inClusters: "0x26,0x33", deviceJoinName: "Z-Wave RGBW Bulb"
+
+		// GENERIC_TYPE_SWITCH_MULTILEVEL:SPECIFIC_TYPE_COLOR_TUNABLE_MULTILEVEL
+		// dimmable, color control
+		fingerprint deviceId: "0x1102", inClusters: "0x26,0x33", deviceJoinName: "Z-Wave RGBW Bulb"
+
+		// GENERIC_TYPE_SWITCH_MULTILEVEL:SPECIFIC_TYPE_POWER_SWITCH_MULTILEVEL
+		// dimmable, color control
+		//
+		// Note: explicitly delimiting application command classes as secure with 0xF1,0x00
+		fingerprint deviceId: "0x1101", inClusters: "0xF1,0x00,0x26,0x33", deviceJoinName: "Z-Wave RGBW Bulb"
```

Comment:
@msandstedt, @gausnes mentioned that the fix for HCS-1050 will be available in Prod environment by two weeks. The changes to these DTHs will more than likely be available in prod in two weeks as well, so I was thinking we can leave these workarounds out since the issues exist in the prod as of now. Thoughts?

Reply:
Makes sense.  I will strip these out and then rebase.  I believe this will invalidate your approval - sorry for the churn!

Almost done here...

Reply:
OK, I pushed up what I consider final (apologies - this invalidated the current approvals).

I believe this addresses everything discussed with the reviewers.  Please note that aeon-led-bulb.groovy now actually contains no fingerprints.  I believe this is appropriate because in fact, Aeon bulbs are already matching to the rgbw-light.groovy with these existing fingerprints:

* fingerprint mfr: "0086", prod: "0003", model: "0062", deviceJoinName: "Aeotec LED Bulb" //EU
* fingerprint mfr: "0086", prod: "0103", model: "0062", deviceJoinName: "Aeotec LED Bulb" //US

I verified that no Aeon bulbs have matched to aeon-led-bulb.groovy in the last 30 days.  The only bulbs that have matched there will be better controlled with rgbw-light.groovy.